manifests/h/HoYoverse/GenshinImpact/2.25.2.0/HoYoverse.GenshinImpact.installer.yaml

```diff
@@ -0,0 +1,12 @@
+PackageIdentifier: HoYoverse.GenshinImpact
+PackageVersion: 2.25.2.0
+Installers:
```

Comment:
This application may also have a dependency, which isn't supported yet. I'm not sure if an unattended install would hide this prompt
![image](https://user-images.githubusercontent.com/12611259/232563480-4c231eb3-3bf9-4ea4-bdd2-5b6bc7de7719.png)